src/systems/filecoin_mining/storage_mining/storage_miner_actor_code.go

```diff
@@ -280,8 +280,7 @@ func (a *StorageMinerActorCode_I) _onSuccessfulPoSt(rt Runtime, postSubmission p
 	for _, sectorNo := range st.Impl().ProvingSet_.SectorsOn() {
 		sectorState, found := st.Sectors()[sectorNo]
 		if !found {
-			// TODO: determine proper error here and error-handling machinery
-			rt.Abort("Sector state not found in map")
+			rt.AbortStateMsg("Sector state not found in map")
```

Comment:
Could you not argue this is an ArgMsg?

Reply:
I agree the distinction is somewhat arbitrary. In general, my heuristic has been to use ArgMsg only when the relevant value is an argument to the public method invocation, and use StateMsg for other internal precondition violations (even if they are "arguments" to an internal method). But I think this is easy enough to change in a followup PR. (cc @zixuanzh who may have more clarity on the intended semantics of the actor code here)